Mlem/Views/Tabs/Feeds/Feed View.swift

```diff
@@ -53,18 +53,21 @@ struct FeedView: View {
     @State var isLoading: Bool = false
     @State var shouldLoad: Bool = false
     
+    @AppStorage("isTranslucentBar") var isTranslucentBar: Bool = true
+    
     // MARK: - Main Views
     
     var body: some View {
         contentView
             .frame(maxWidth: .infinity, maxHeight: .infinity)
-            .background(Color.secondarySystemBackground)
+            .background(isTranslucentBar ? Color.secondarySystemBackground : Color.systemBackground)
```

Comment:
I don't think this should change--the `.secondarySystemBackground` is a good visual indicator that a space is "non-content." Things like the refresh and the end-of-feed indicator don't look quite right on default background.

Reply:
I think it makes for a cleaner look with the adjusted background, especially during the start up: [with changes](https://github.com/mlemgroup/mlem/assets/20093825/2db91c87-e711-45e6-8c6e-cf89fd432a90) | [without changes](https://github.com/mlemgroup/mlem/assets/20093825/ec13ba43-4f45-4e7f-85a0-189542f83bc4) (less pop-in). It also fits the same aesthetic that an expanded post has (it's also the same color on top and on the bottom there. 

The top of the feed always has the progress indicator, while the bottom is probably never reached anyways. I understand what you mean, though. 

Have I managed to change your mind or do you still prefer the `.secondarySystemBackground`? I'll change it if you still prefer that, of course.

Reply:
We can merge it as-is--you're right that it's not consistent across the app, though, which should probably be addressed. I've opened a followup #493 to fix that, we can revisit this then.

Reply:
Nice :) 
Maybe it should be the same background regardless of the transparency then? With the transparency it fits better in but  it's the same inconsistency